cli/internal/oauth/manager/manager.go

```diff
@@ -93,7 +93,8 @@ func (m *OAuthManager) LoginDevice(ctx context.Context, w io.Writer) (*types.Aut
 	}
 
 	_, _ = fmt.Fprintln(w, aec.Bold.Apply("\nUSING WEB-BASED LOGIN"))
-	_, _ = fmt.Fprintln(w, "To sign in with credentials on the command line, use 'docker login -u <username>'")
+	_, _ = fmt.Fprintln(w, "> To sign in with username and PAT credentials on the command line, use 'docker login -u <username>'")
+	_, _ = fmt.Fprintln(w, "> When prompted to enter a password, enter the PAT retrieved from: "+aec.Underline.Apply("https://app.docker.com/settings"))
```

Comment:
??? This feels like the CLI going around the CLIs back and telling users to do something else 😅 

Users _can_ use PATs, but telling users explicitly to use PATs from the Web login prompt feels weird when they can also use passwords. And it turns an already long prompt that most users don't read all the way through even longer.

Reply:
thoughts @dvdksn ?

Reply:
Yeah, making this prompt even longer will have the opposite effect I think. The longer it is, the less likely a user is to read it at all. That's why we were trying to keep it as terse as possible in the first iteration.